### PR TITLE
NFT Section added in README.md

### PR DESCRIPTION
**Pull request by jyotirmoydotdev**

### CODE REVIEW

**Review by mrhrifat — CHANGES_REQUESTED** [review]
If you think to create a `group` of badge, it shortcut option should at top `Menu` section. Please make a change to `NFT` **second** badge `logoColor` to white.

**Comment by jyotirmoydotdev** [comment]
Changes done ✅

File: README.md [hunk 1]
```diff
@@ -10,65 +10,62 @@ Improve your README.md profile with these amazing badges. See a sample [here](ht
...
+  - [❤ Funding 🔝](#-funding-)
+  - [🎮 Games 🔝](#-games-)
+  - [🤜 Group 🔝](#-group-)
+  - [👩‍💻 IDE 🔝](#-ide-)
+  - [🔌 IDE Plugin 🔝](#-ide-plugin-)
+  - [👩‍💻 Languages 🔝](#-languages-)
+  - [🧐 Linters 🔝](#-linters-)
+  - [🎡 Low Code Platforms 🔝](#-low-code-platforms-)
+  - [📱 Mobile Frameworks 🔝](#-mobile-frameworks-)
+  - [🏞️ NFT 🔝](#️-nft-)
```
Comment (jyotirmoydotdev) [inline README.md:39]: Shortcut to NFT section

**Review by mrhrifat — CHANGES_REQUESTED** [review]
Please don't make any change to `Quicke Navigate` of section `Menu`. If you've any idea than share it first by creating `Issue`. Revert & just add `NFT`.

**Pull request closed without merging**